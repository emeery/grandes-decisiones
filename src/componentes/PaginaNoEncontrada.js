import React from 'react';
import {Link} from 'react-router-dom';
const PaginaNoEncontrada = () => (
    <div>
        <h2>404 - UPS!</h2>
        <h3>No es lo que buscas?</h3>
        <Link to='/'>a casa</Link>
    </div>
); export default PaginaNoEncontrada;