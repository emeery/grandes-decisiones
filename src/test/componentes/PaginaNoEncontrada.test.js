import React from 'react';
import { shallow } from 'enzyme';
import PaginaNoEncontrada 
from '../../componentes/PaginaNoEncontrada';
test('el componnte PaginaNoEncontrada',()=>{
    const envoltura = shallow(<PaginaNoEncontrada/>);
    expect(envoltura).toMatchSnapshot();
});