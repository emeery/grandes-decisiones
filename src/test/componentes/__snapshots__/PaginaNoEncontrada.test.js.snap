// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`el componnte PaginaNoEncontrada 1`] = `
<div>
  <h2>
    404 - UPS!
  </h2>
  <h3>
    No es lo que buscas?
  </h3>
  <Link
    replace={false}
    to="/"
  >
    a casa
  </Link>
</div>
`;
